agent/ansible/Makefile

```diff
@@ -18,6 +18,7 @@ test:   build
 	ansible-galaxy collection install ${tb} -p ./collections
 	git submodule update --init --recursive
 	cd ${colldir}; ansible-test sanity
+	# test for README.md in *every* role
```

Comment:
Sounds like a `FIXME`? Is this still necessary?

Reply:
Yes, it is a FIXME. The problem is that Galaxy imposes rules and regulations, but `ansible-test sanity` is unaware of them, so it does not flag them. They are found when the collection is uploaded and Galaxy goes through its steps, and fails (with messages that sometimes require tea leaf reading abilities). So I would like to replace the comment with a real test, but I didn't want to do it while running on empty. I can probably take a stab at implementing the test now, but I don't want to delay merging the PR for this. I'll open a (decidedly minor) issue and fix it later.

Reply:
See issue #2595

Reply:
This comment is problematic:  it's describing a function or code which _ought_ to be here but isn't.  This would be made clear if it were prefixed with `FIXME` or `TODO`, but I believe we have a preference for removing those prior to merging.  So, if we're going to retain this comment in the code, it really should be enhanced (e.g., replace "test" with "This code *should* be testing").